webpack.config.js

```diff
@@ -66,19 +65,13 @@ const config = {
     noParse: /node_modules\/sqlite-parser\/dist\/sqlite-parser-min.js/,
     rules: [
       {
-        test: /\.(png|gif)$/,
-        use: { loader: "url-loader?name=[name]@[hash].[ext]&limit=6000" },
-      },
-      {
-        test: /\.(pdf|ico|jpg|svg|eot|otf|woff|woff2|ttf|mp4|webm)$/,
-        use: {
-          loader: "file-loader",
-          options: {
-            name: "[name]@[hash].[ext]",
-            useRelativePath: true,
-          },
+        test: /\.(pdf|png|gif|ico|jpg|svg|eot|otf|woff|woff2|ttf|mp4|webm)$/,
+        type: "asset/resource",
+        generator: {
+          filename: "[name]@[hash][ext]",
```

Comment:
did you mean to exclude the `.ext` in `filename: "[name]@[hash][ext]",`?

Reply:
The `.` seems to be unnecessary in webpack 5's builtin functionality vs. what was necessary in the older file-loader config.

The files seem to come out appropriately:
<img width="588" alt="Screenshot 2023-04-13 at 12 06 57 PM" src="https://user-images.githubusercontent.com/575602/231858922-3d99e15e-014f-45f6-a136-b46dbdcddadf.png">